apps/devportal/src/components/common/contribute.tsx

```diff
@@ -0,0 +1,32 @@
+import { PageInfo } from '@/src/lib/interfaces/page-info';
+import { Card, CardBody, Heading, Link, Text } from '@chakra-ui/react';
+
+type GithubContributionNoticeProps = {
+  pageInfo: PageInfo;
+};
+
+const GithubContributionNotice = ({ pageInfo }: GithubContributionNoticeProps) => {
+  
+  console.log(pageInfo);
```

Comment:
Can you remove this console.log line as its going to clog up the server logs?

Reply:
Done